### PR TITLE
feat: split api status checks

### LINKED ISSUE #19: Node API check should not be part of first check
With the latest changes to the debug API (not yet released), some endpoints now start even if all the conditions are not met (unlike the node API). The Node API check should probably be one of the last as it needs pretty much all the previous steps to start.

### PR DESCRIPTION
**Pull request by matmertz25**
resolves #19

### CODE REVIEW

**Review by vojtechsimetka — APPROVED** [review]

**Review by vojtechsimetka — CHANGES_REQUESTED** [review]
I thought this is just a commit lint problem but there are build issues that need to be fixed before merging.

**Review by vojtechsimetka — APPROVED** [review]
Good from my side, but I'd like @agazso eye as I fixed some of the issues

File: src/pages/status/SetupSteps/DebugConnectionCheck.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                  </AccordionDetails>
+                </Accordion>
+              </Typography>
+            :
+            null}
+            </div>
+          </div>
+    </div>
+    )
+}
```
Comment (agazso) [inline src/pages/status/SetupSteps/DebugConnectionCheck.tsx:76]: No newline at end of file (linter will fix it)

**Review by agazso — APPROVED** [review]
Looks good, I only found formatting issues that will be hopefully solved by the linter config

**Pull request merged**